packages/commonwealth/client/scripts/views/components/react_quill_editor/QuillLinkModal.tsx

```diff
@@ -42,7 +42,7 @@ export const AddLinkModal = ({
           <CWModalBody>
             <CWTextInput
               label="Link Text"
-              placeholder="Enter Link Text"
+              placeholder="Enter link text"
```

Comment:
@MuhammadKaleem171 Should be "Link text", only the first letter of the first word

Reply:
i have changed the Link Text to link text